pegasus/sites.v3/code.org/views/curriculum_pages_additional_resources.haml

```diff
@@ -0,0 +1,41 @@
+:ruby
+  additional_resources = [
+    {
+      title: hoc_s('curricula_pages.additional_resources.catalog.title'),
+      image: "/images/course-catalog-illustration.png",
+      desc: hoc_s('curricula_pages.additional_resources.catalog.desc'),
+      url: CDO.studio_url("/catalog"),
+      button_label: hoc_s('curricula_pages.additional_resources.catalog.button')
+    },
+    {
+      title: hoc_s('curricula_pages.additional_resources.self_paced.title'),
+      image: "/images/professional-learning/self-paced-pl-discover.png",
+      desc: hoc_s('curricula_pages.additional_resources.self_paced.desc'),
+      url: "/online-pl",
+      button_label: hoc_s('curricula_pages.additional_resources.self_paced.button')
+    },
+    {
+      title: hoc_s('curricula_pages.additional_resources.workshops.' + (is_elementary_page ? 'title_k_5' : 'title_6_12')),
+      image: "/images/professional-learning/k-12-workshops.png",
+      desc: hoc_s('curricula_pages.additional_resources.workshops.desc'),
+      url: is_elementary_page ? "/professional-learning/elementary" : "/professional-learning/middle-high",
+      button_label: hoc_s('curricula_pages.additional_resources.workshops.button'),
+    },
+  ]
+
+%h2.centered
+  =hoc_s('curricula_pages.additional_resources.title')
+%p.centered
+  =hoc_s('curricula_pages.additional_resources.desc')
+.action-block__wrapper.action-block__wrapper--three-col
+  - additional_resources.each do |resource_item|
+    .action-block.action-block--one-col.flex-space-between
```

Comment:
Add the `.white` class here so these show up better on the gray bg. 
```suggestion
    .action-block.action-block--one-col.flex-space-between.white
```

Reply:
Ah thank you for catching this! This was on my checklist but I totally forgot it